### ISSUE TITLE
Response status code does not indicate success: 302 (Moved Temporarily)

### ISSUE THREAD

**Issue by schoenm1** [lordmilko/PrtgAPI#385]
### Describe the bug

If I want to set ObjectProperty ``-comments "TEST comment"``, the comment will be set correctly on PRTG server.
But If PRTGClient Version is 0.9.18 or newer, following error occur:
```
 Get-Sensor -id 40469 | Set-ObjectProperty -comments "TEST comment" -verbose
VERBOSE: Performing the operation "Set-ObjectProperty Comments = 'TEST comment'" on target "'FW Version' (ID: 40469)".
VERBOSE: Set-ObjectProperty: Synchronously executing request
https://prtgserver.domain.com/editsettings?id=40469&comments_=TEST+comment&nosession=1&username=myuser&passhash=***********
Set-ObjectProperty : Response status code does not indicate success: 302 (Moved Temporarily).
At line:1 char:24
+ Get-Sensor -id 40469 | Set-ObjectProperty -comments "TEST comment"
+                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Set-ObjectProperty], HttpRequestException
    + FullyQualifiedErrorId : System.Net.Http.HttpRequestException,PrtgAPI.PowerShell.Cmdlets.SetObjectProperty
```
If same command is executed with PrtgAPI Version <=0.9.17, no error.

### Steps to reproduce

```powershell
Connect-PrtgServer myprtg.domain.com
 Get-Sensor -id 40469 | Set-ObjectProperty -comments "TEST comment"
```

### What is the output of 'Get-PrtgClient -Diagnostic'?

```powershell
- Not working on Version 0.9.18 + 0.9.19
- Command working without 302 (Moved Temporarily) error with <=0.9.17

PSVersion      : 5.1.22621.2506
PSEdition      : Desktop
OS             : Microsoft Windows 11 Enterprise
PrtgAPIVersion : 0.9.18
Culture        : de-CH
CLRVersion     : 533320
PrtgVersion    : 23.4.90.1299
PrtgLanguage   : english.lng
```

### Additional context

In Browser development mode, I see the 302 (temporarily moved) code, if I enter
https://prtgserver.domain.com/editsettings?id=40469&comments_=TEST+comment&nosession=1&username=myuser&passhash=***********
![image](https://github.com/lordmilko/PrtgAPI/assets/1053744/912f04c8-2b4e-498e-923f-05d7a9b7a679)
After setting the comment, redirect to https://prtgserver.domain.com/sensor.htm?id=40469&tabid=1

**Comment by lordmilko** [comment]
PrtgAPI 0.9.18 adds an additional parameter `nosession=1` when executing `/editsettings` as recommended by Paessler to prevent memory leaks on the PRTG Core Server.

* Regardless of the fact the request returns a 302 response, is the comment actually updated?
* If you execute the API request manually without specifying `nosession=1`, does it work?

**Comment by schoenm1** [comment]
I've done some more testing.

- Regardless of the fact the request returns a 302 response, is the comment actually updated? --> Yes, comment will always be updated, regardless of whether error ``302 (Moved Temporarily)`` is displayed.
- Regardless, if I connect directly to my PRTG Core Server or from WAN with LoadBalanced, same behavior
- on all tested PRTG Server with Version 24.1.92.1554+: no error, regardless if using 0.9.17 or 0.9.19
- on all tested PRTG Server with Version 24.1.90.1306+: no error with 0.9.17, but error ``302 (Moved Temporarily)`` with 0.9.19

I think you can close this 'alleged-bug'. If PRTG Server is updated to newest version, everything is working again.
@lordmilko thanks for your support !

**Comment by lordmilko** [comment]
Thanks @schoenm1,

The conclusion of this issue is that this issue was introduced around PRTG version 23.4.90.1299 and is resolved in PRTG 24.1.92.1554
